osu.Game/Database/RealmWriteUsage.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using Realms;
+
+namespace osu.Game.Database
+{
+    public class RealmWriteUsage : IDisposable
```

Comment:
Again, this is drawing from our existing implementation. I'm not sure this is required or if we should go for something more simple, but leaving it as-is for now (usage is very minimal).